src/index.d.ts

```diff
@@ -426,6 +501,67 @@ declare namespace SDKStandardComponents {
          */
         stop(): void
     }
+    namespace Logger {
+        import safeStringify from 'fast-safe-stringify'
+        type Level = 'verbose' | 'debug' | 'warn' | 'error' | 'trace' | 'info' | 'fatal'
+        type TimestampFormatter = (ts: Date) => string;
+        type Stringify = (toBeStringified: unknown) => string;
+        type LoggerStringify = ({ ctx: unknown, msg: unknown, level: Level }) => string
+        type BuildStringify = ({ 
+            space: number,
+            printTimestamp: boolean,
+            timestampFmt: TimestampFormatter,
+            stringify: Stringify 
+        }) => LoggerStringify;
+
+        function buildStringify({
```

Comment:
I think this should have a return signature, which I think will just be `LoggerStringify` like you have above